style.css

```diff
@@ -1,9 +1,331 @@
+body {
+    font-family: 'Roboto', sans-serif;
+    color: #1C1D21;
+}
+/* TODO: never set exact width on body */
 
-.header{
-    display:flex;
+.header {
+    display: flex;
+    height: 30px;
+}
+
+.nav {
+    display: flex;
     flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
 }
-.nav{
+
+.link_flex_container {
+    display: flex;
+    justify-content: flex-end;
+    width: 80%;
```

Comment:
do you need percentage value in here?

Reply:
instead of setting width for this element, set margins and paddings for it's children so width will be calculated automatically